manim/utils/testing/_frames_testers.py

```diff
@@ -48,6 +49,22 @@ def check_frame(self, frame_number: int, frame: np.ndarray):
             )
             self._frames_compared += 1
         except AssertionError as e:
+            number_of_matches = np.isclose(
+                frame, self._frames[frame_number], atol=1.01
+            ).sum()
+            number_of_mismatches = frame.size - number_of_matches
+            print(number_of_mismatches / frame.size)
```

Comment:
```suggestion
```

Reply:
Ah, read your comments in the wrong order, sorry!